scico/optimize/admm.py

```diff
@@ -481,7 +481,7 @@ def __init__(
 
         # dynamically create itstat_func; see https://stackoverflow.com/questions/24733831
         itstat_return = "return(" + ", ".join(["obj." + attr for attr in itstat_attrib]) + ")"
-        scope = {}
+        scope: dict[str, Callable] = {}
```

Comment:
Same as above. In line 484 dict[str, Any] should be of type Dict of the typing module.

Reply:
As above.